src/pages/lib-v1/auth/emailpassword/q/platform/[platform].mdx

```diff
@@ -258,19 +237,31 @@ async function signIn() {
 </Block>
 </BlockSwitcher>
 
+The `signIn` API response will include a `nextStep` property, which can be used to determine if further action is required. It may return the following next steps:
+
+- `CONFIRM_SIGN_IN_WITH_NEW_PASSWORD_REQUIRED` - The user's password must be reset before the sign in can be completed.
```

Comment:
Here the user is forced to create a new password after an admin has created an user with a temporary password.